LEAF_Request_Portal/sources/Form.php

```diff
@@ -3712,14 +3703,23 @@ public function query(string $inQuery): mixed
 									WHERE format = 'orgchart_employee') rj_OCEmployeeData ON (lj_data.indicatorID = rj_OCEmployeeData.indicatorID) ";
         }
 
-        if ($joinInitiatorNames)
-        {
-            $joins .= "LEFT JOIN (SELECT userName, lastName, firstName FROM {$this->oc_dbName}.employee) lj_OCinitiatorNames ON records.userID = lj_OCinitiatorNames.userName ";
-        }
+        //backwards compat: userMetadata properties are empty for accounts that were inactive when prior md values were updated.
+        //userMetadata alternatives here prevent the initiator field from displaying 'null, null' if metadata is empty.
+        //Handled here due to high customization of view_reports, view_search and other reports
+        $resSQL = 'SELECT *,
```

Comment:
This concerns me.  Not because of the select *, but because of the amount of processing it has to do on each line.  How much is this going to slow down db responses, especially during the middle of the day?  It might be better to do the pull for * first, process it on the machine designed for these operations, and then reach back out for the data it needs in addition.

Reply:
It looks like it might have added 10-25% to a 'select * ' query.  Going to rework this to keep the initiatorNames "join" logic, but only use it in a backwards compatible way to add the additional SQL if it's needed.  This will keep queries where initiator information is not requested the same speed, but still use json instead of the join when it is (where it should be considerably faster)